packages/swingset-theme-hashicorp/src/components/side-nav/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { NavigationTree } from 'swingset/types'
+import Category from './category'
+
+type SideNavBarProps = {
+  categories: NavigationTree
+}
+
+function SideNavigation(props: SideNavBarProps) {
+  const { categories } = props
+
+  
+
+  const categoriesJSX = Object.values(categories).map(
+    (category) => (
+      <Category title={category.title} key={category.title} items={category.children} />
+    )
+  )
```

Comment:
I _think_ `Object.values()` will be unnecessary here if we fix the return value of `getNavigationTree`. In that case, we can use `categories.map()`.